core/src/apps/ethereum/sign_tx.py

```diff
@@ -2,13 +2,23 @@
 
 from trezor.crypto import rlp
 from trezor.messages import EthereumTxRequest
+from trezor.utils import BufferReader
 from trezor.wire import DataError
 
+from apps.ethereum import staking_tx_constants as stx
```

Comment:
I would consider importing it `as constants`, so all the code reads `constants.XYZ`

Reply:
https://github.com/trezor/trezor-firmware/pull/3534/commits/b3eb30b259e2034a10e613d2f20c7761538a91f7